frontend/public/components/api-explorer.tsx

```diff
@@ -644,6 +645,19 @@ const APIResourcePage_ = ({
     },
   ];
 
+  const namespace = kindObj.namespaced ? match.params.ns : null;
```

Comment:
You might want to make this `undefined` because `useAccessReview` sets a default value only when undefined.

```suggestion
  const namespace = kindObj.namespaced ? match.params.ns : undefined;
```